src/utils/Validator.ts

```diff
@@ -0,0 +1,13 @@
+export class Validator {
+  // tslint:disable-next-line: no-empty
+  public constructor() {}
+
+  /**
+   * Validate query or path parameter provided in the request
+   * @param parameter Query or path parameter provided
+   * @returns boolean Result of logic determining validity
+   */
+  public parameterIsValid(parameter: string): boolean {
+    return parameter ? parameter !== "undefined" && parameter !== "null" : false;
```

Comment:
Is the value of event.pathParameters coming through as strings even when the value is undefined or null?

Reply:
You should be able to just check if event.pathParameters is falsy (undefined or null) instead of needing this validator class

Reply:
I've copied this method over from the activities service where it is used when checking for both path & query parameters. From testing yes, null and undefined when it's pulled from the path parameter is always a string value:   PATH_PARAMS: '{"id":"undefined"}'.